src/docs/user/references/Notification-service.adoc

```diff
@@ -116,6 +132,69 @@ and activate the notification method to start receiving notification by this mea
 If a notification method parameter is deleted or changed and reused in the future, then a new confirmation
 request will be triggered.
 
+[[channels_section]]
+=== Channels
+
+==== Principle
+
+A notification service <<_glossary_notification_channel, Channel>> is a centralized space in the <<_glossary_notification_portal, Notification Portal>>
+for groups of users to receive notifications created from a workflow using the Channel_Notification task.
+It consists of a unique name and a set of user groups authorized to subscribe to the channel.
+
+In order to receive channel notifications it is required to create one or more channels and update the channel
+<<channels_subscription,*Channel subscription*>> to specify which channels we wish to be notified of.
+
+==== Rules
+
+- Unlike other notifications, channel notifications are retroactive. It means that when a user subscribes to a channel that already had notifications
+, they will appear to the user.
+- Channel notifications cannot be deleted, the link:../admin/ProActiveAdminGuide.html#_notification_housekeeping[Housekeeping]
+takes care of it once their life span is exceeded.
+- Notification service administrators can subscribe to any channels. Other users must be part of at least one of the authorized groups
+defined in the channel in order to subscribe to a channel, unless all groups are authorized.
+
+==== Channel Creation
+
+All users are able to create channels for any group. It is required to provide a unique channel name and a list of groups
+which will be able to subscribe to the channel.
+
+It can either be authorized for all groups, one or more groups of the current users, or any free input.
+
+[title="Channel creation"]
+image::create-channel.png[align="center"]
+
+==== Channel Management
+
+Channel management is where users can update channels by modifying a channel's name, add or remove authorized groups or delete it.
+
+A notification service administrator is able to update any channels, whether he is the creator of the channel or not.
+Other users can update only channels that they have created.
+
+To update a channel's name or authorized groups, simply make the update and click on the __Save__ button.
+
+To delete a channel, click on the __Bin__ icon on the left of the channel definition.
+
+[title="Channel creation"]
+image::channel-management.png[align="center"]
+
+==== Channel Notification
+
+In order to send a channel notification a user must append the Channel_Notification task to a <<_glossary_workflow, Workflow>>.
```

Comment:
In order to send a channel notification, (missing comma)
Add backticks around Channel_Notification

a user must add